src/it/java/net/spy/memcached/ProtocolBaseCase.java

```diff
@@ -155,6 +155,12 @@ public void testDelayedFlush() throws Exception {
       restoreClusterConfig(current_config, endpoints);
       Thread.sleep(1000); // Wait for the config to restore
     }
+
+    // Restore the server configuration for TLS enabled Dynamic mode after flush
+    if (TestConfig.getInstance().getClientMode() == ClientMode.Dynamic && TestConfig.isTlsMode()) {
+      restoreClusterConfigForTLS(TestConfig.PORT_NUMBER);
+    }
```

Comment:
Can you simply handle TLS in `restoreClusterConfig()` above rather than adding this new code?

Reply:
It will never revoke `restoreClusterConfig()` method if `!TestConfig.getInstance().getEngineType().isSetConfigSupported()` is false. So I decide to make it separate from `restoreClusterConfig()`. I will rename `restoreClusterConfigForTLS()` to `setClusterConfigForTLS()` to avoid confusion.